src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadCreate.js

```diff
@@ -36,16 +43,21 @@ const AnnotationThreadCreate = ({
     target,
 }: Props) => {
     const [isPending, setIsPending] = React.useState(false);
+    const events = useAnnotatorEvents({ eventEmitter });
 
     const handleCreate = (text: string): void => {
         const { id, permissions = {}, file_version = {} } = file;
+        const requestId = uniqueId('annotation_');
         setIsPending(true);
+        events.emitAddAnnotationStartEvent({ text }, requestId);
+
         if (!currentUser) {
             throw getBadUserError();
         }
 
         const successCallback = (annotation: Annotation) => {
             onAnnotationCreate(annotation);
```

Comment:
It really seems like the consuming component of `AnnotationThread` should be managing the event emitter rather than passing it down to every individual component. It's something we should look to refactor in the future.